CHANGELOG.md

```diff
@@ -4,6 +4,50 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.22.0] - 2022-06-20
+
+With this release, subxt can perform runtime upgrades to ensure that extrinsics are properly constructed for the
+target node. The dependabot is added to the repository to keep the crate dependencies updated.
+
+The API's ergonomics are improved for iterating over N-map storage keys, allowing the API to be cloned between
+runtime tasks, running multiple storage queries without borrowing the client, and dry run extrinsics for
+validation purposes.
+
+This release also improves the subxt's documentation, adds UI testing, and defaults the `subxt-cli` to return metadata
+bytes instead of the JSON format.
+
+### Fixed
+
+- Handle `StorageEntry` empty keys ([#565](https://github.com/paritytech/subxt/pull/565))
+- Fix documentation examples ([#568](https://github.com/paritytech/subxt/pull/568))
+- Fix cargo clippy ([#548](https://github.com/paritytech/subxt/pull/548))
+- fix: Find substrate port on different log lines ([#536](https://github.com/paritytech/subxt/pull/536))
+
+### Added
+
+- Followup test for checking propagated documentation ([#514](https://github.com/paritytech/subxt/pull/514))
+- feat: refactor signing in order to more easily be able to dryrun ([#547](https://github.com/paritytech/subxt/pull/547))
+- added subscribe blocks example ([#521](https://github.com/paritytech/subxt/pull/521))
+- Add subxt documentation ([#546](https://github.com/paritytech/subxt/pull/546))
+- Add GitHub Actions to Dependabot and CODEOWNERS ([#555](https://github.com/paritytech/subxt/pull/555))
```

Comment:
IMO not every single PR since the last release needs to be included, most PRs which are not relevant to lib users can be filtered out, thus imrpoving signal/noise.